test/src/test/java/hudson/LauncherTest.java

```diff
@@ -271,10 +273,10 @@ private void assertMultipleStdioCalls(String message, Node node, boolean emitStd
         assertEquals(message, 0, ps.join());
         if (outputIn2) {
             assertThat(message, baos2.toString(), containsString("hello"));
-            assertThat(message, baos1.toString(), isEmptyString());
+            assertThat(message, baos1.toString(), is(emptyString()));
```

Comment:
There are some * imports that can also be refactored, and `isEmptyString()` matcher can probably be not imported.